watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/peracetic_acid_disinfection/peracetic_acid_disinfection.py

```diff
@@ -0,0 +1,296 @@
+    m.fs.costing.initialize()
+
+    assert_degrees_of_freedom(m, 0)
+    assert_units_consistent(m)
+    results = solve(m)
+    assert_optimal_termination(results)
+    display_costing(m)
+
+    return m, results
+
+
+def build():
+    # flowsheet set up
+    m = ConcreteModel()
+    m.db = Database()
+
+    m.fs = FlowsheetBlock(default={"dynamic": False})
+    m.fs.prop = prop_ZO.WaterParameterBlock(
+        default={
+            "solute_list": [
+                "peracetic_acid",
+                "total_coliforms_fecal_ecoli",
+            ]
+        }
+    )
+
+    # unit models
+    # feed
+    m.fs.feed = FeedZO(default={"property_package": m.fs.prop})
+
+    # PAA treatment
+    m.fs.PAA = PeraceticAcidDisinfectionZO(
+        default={
+            "property_package": m.fs.prop,
+            "database": m.db,
+        },
+    )
+
+    # product stream
+    m.fs.treated_water = Product(default={"property_package": m.fs.prop})
+
+    # connections
+    m.fs.s01 = Arc(source=m.fs.feed.outlet, destination=m.fs.PAA.inlet)
+    m.fs.s02 = Arc(source=m.fs.PAA.treated, destination=m.fs.treated_water.inlet)
+
+    # expand arcs
+    TransformationFactory("network.expand_arcs").apply_to(m)
+
+    # scaling
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def set_operating_conditions(m):
+    # ---specifications---
+    # feed
+    flow_vol = 240000 * pyunits.m**3 / pyunits.day
+    conc_mass_PAA = 1.8 * pyunits.mg / pyunits.liter
+    conc_mass_ecoli = 2.0e-4 * pyunits.mg / pyunits.liter
+    m.fs.feed.flow_vol[0].fix(flow_vol)
+    m.fs.feed.conc_mass_comp[0, "peracetic_acid"].fix(conc_mass_PAA)
+    m.fs.feed.conc_mass_comp[0, "total_coliforms_fecal_ecoli"].fix(conc_mass_ecoli)
+
+    solve(m.fs.feed)
+
+    m.fs.PAA.load_parameters_from_database(use_default_removal=True)
+
+
+def initialize_system(m):
+    seq = SequentialDecomposition()
+    seq.options.tear_set = []
+    seq.options.iterLim = 1
+    seq.run(m, lambda u: u.initialize())
+
+
+def solve(blk, solver=None, tee=False, check_termination=True):
+    if solver is None:
+        solver = get_solver()
+    results = solver.solve(blk, tee=tee)
+    if check_termination:
+        assert_optimal_termination(results)
+    return results
+
+
+def display_results(m):
+    print("\n+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+    print("++++++++++++++++++++ DISPLAY RESULTS ++++++++++++++++++++")
+    print("+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+
+    unit_list = ["feed", "PAA"]
+    for u in unit_list:
+        m.fs.component(u).report()
+
+    print("\n+++++++++++++++++++++++++++++++++++++++++++++++++++++++++")
+
+
+def add_costing(m):
+    source_file = os.path.join(
+        os.path.dirname(os.path.abspath(__file__)),
+        "peracetic_acid_case_study.yaml",
+    )
+
+    m.fs.costing = ZeroOrderCosting(default={"case_study_definition": source_file})
+
+    costing_kwargs = {"default": {"flowsheet_costing_block": m.fs.costing}}
```

Comment:
```suggestion
    costing_kwargs = {"flowsheet_costing_block": m.fs.costing}
```
This should get rid of the deprecation warning

Reply:
Addressed in latest commit.